account-service/src/main/java/com/bdonor/accountservice/Controller/AccountController.java

```diff
@@ -17,74 +15,57 @@
     @Autowired
     private AccountHelper Service_functions;
 
-    private UserRepository UserRepo;
-
-    public AccountController(UserRepository userRepo) {
-        UserRepo = userRepo;
-    }
-
-    @ResponseBody
-    @RequestMapping("/CreateTwo") // Works
-    public String CreateTwo(@RequestParam String firstname) {
-        User CreateUser = Service_functions.createNew(firstname);
-        return firstname + " Added";
-    }
-
-    @ResponseBody
-    @RequestMapping("/testController") // Works
-    public String testResponse() {
-        
-        return "Response: OK";
-    }
-    @ResponseBody
-    @GetMapping(value = "/create/{bloodGroup}/{firstname}/{_surname}/{_email}/{_addressline}/{_postcode}/{_password}") // Works
+    @ResponseBody // Works
+    @GetMapping(value = "/create/{bloodGroup}/{firstname}/{_surname}/{_email}/{_password}/{_addressline}/{_password}")
     public String create( @PathVariable String bloodGroup , @PathVariable  String firstname, @PathVariable  String _surname, @PathVariable  String _email, @PathVariable  String _password, @PathVariable  String _addressline, @PathVariable  String _postcode){
         User CreateUser = Service_functions.create(bloodGroup, firstname,  _surname,  _email,  _password,  _addressline,  _postcode);
         System.out.println("this is working");
         return CreateUser.toString();
     }
 
     @ResponseBody
-    @GetMapping("/getUser/{firstName}") // now OK
-    public String getUser( @PathVariable String firstName ){
+    @GetMapping("/getUser/{firstname}") // Works Partially - Only works for one user within database, if there are more with the same name, error is given
+    public String getUser( @PathVariable String firstname ){
         System.out.println("Working");
-        return Service_functions.getByfirstName(firstName).toString();
+        return Service_functions.getByfirstName(firstname).toString();
     }
 
     @ResponseBody
     @GetMapping("/getAll")
-    public String getAllUsers(){ // THIS DOESNT WORK
+    public String getAllUsers(){ // Works
         System.out.println("This Works");
         return Service_functions.getAll().toString();
     }
 
     @ResponseBody
-    @RequestMapping("/updateUser/{bloodGroup}/{firstname}/{_surname}/{_email}/{_addressline}/[_postcode")
+    @RequestMapping("/updateUser/{bloodGroup}/{firstname}/{_surname}/{_email}/{_password}/{_addressline}/{_password}") // Works Partially - Creates new user instead of updating current
     public String updateUser( @PathVariable String bloodGroup , @PathVariable  String firstname, @PathVariable  String _surname, @PathVariable  String _email, @PathVariable  String _password, @PathVariable  String _addressline, @PathVariable  String _postcode ){
```

Comment:
parameters inconsistent, let's use "_" in class variables only (it's only naming conventions but goes well with Java)